samples/expensive-test/import-data-text-sentiment-analysis.test.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const path = require('path');
+const {assert} = require('chai');
+const {describe, it} = require('mocha');
+
+const cp = require('child_process');
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+const cwd = path.join(__dirname, '..');
+
+const datasetId = '5148529167758786560';
+const gcsSourceUri =
+  'gs://cloud-ml-data/NL-sentiment/crowdflower-twitter-claritin-80-10-10.csv';
```

Comment:
I see two separate buckets being used for the source of data here.  I worry about proliferation of source buckets.  Can we get these all together?

Reply:
I see 4 different buckets now!

Reply:
Maybe, but not with this PR. These resources are all matching the corresponding samples and tests in the uCAIP python repo, so if we do consolidate them all into one bucket, that should be a little project in its own right.

Reply:
SGTM